buildx.sh

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025 Mastercard
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+cleanup() {
+  echo "Caught SIGINT. Exiting..."
+  exit 1
+}
+
+trap cleanup SIGINT  # Handle SIGINT (CTRL-C)
+
+PACKAGE="pkcs11-tools"
+GITHUB_REPO="https://github.com/Mastercard/pkcs11-tools"
+GITHUB_REPO_COMMIT="HEAD"
+SUPPORTED_ARCHS="amd64 arm64"
+SUPPORTED_DISTROS="ol7 ol8 ol9 deb12 ubuntu2204 ubuntu2404 amzn2023 alpine321"
```

Comment:
Looks like Ubuntu 20.04 is still in LTS (assuming that was the move here since 22.04 and 24.04 are in LTS). Are you looking to cover the full breadth of LTS releases?

https://releases.ubuntu.com/

Reply:
Good catch. Let me see how it works with 20.04.